composer.json

```diff
@@ -1,15 +1,20 @@
 {
 	"name": "silverstripe/mathspamprotection",
 	"description": "This module provides a simple math protection mechanism for prevent spam from your forms.Includes an EditableMathSpamField to integrate with the UserForms module.",
-	"type": "silverstripe-module",
+	"type": "silverstripe-vendormodule",
 	"keywords": ["silverstripe", "spamprotection", "mathspamprotection"],
 	"authors": [{
 		"name": "Will Rossiter",
 		"email": "will@fullscreen.io"
 	}],
 	"minimum-stability": "dev",
 	"require": {
-		"silverstripe/spamprotection": "^2"
+		"silverstripe/spamprotection": "^3"
 	},
+    "autoload": {
+        "psr-4": {
+            "SilverStripe\\SpamProtection\\Maths\\": "code/"
```

Comment:
Sorry, one last thing - can you please update all of these namespaces to `SilverStripe\MathSpamProtection`? The PSR-4 convention is `Vendor\Package`, and we don't like to pollute namespaces across modules.

Reply:
I guessed the wrong convention *sad face*.  Will update after lunch